App.js

```diff
@@ -1,19 +1,24 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, View } from 'react-native';
+import Amplify from 'aws-amplify'
+import awsconfig from './src/aws-exports'
+import Home from './src/Home';
+import Header from './src/Header';
 
+Amplify.configure(awsconfig)
 export default function App() {
   return (
     <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
+      <Header />
+      <Home />
       <StatusBar style="auto" />
     </View>
   );
 }
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
```

Comment:
Why are you removing the `flex` property?

Reply:
I will be adding a ScrollView inside the Home Component in next commit. So, flex property around a ScrollView doesn't make sense.

Reply:
Might be missing something, but ideally the containing `View` of an `App` should fill the available space of the device screen

Reply:
Thanks Caleb, I have added flex to the root Component and justified the content to `flex-start`.

Reply:
**Maybe** we can just remove the `alignItems` and `justifyContent` properties altogether?

Reply:
I have pushed the new commit. It doesn't affect the styling at the moment.